portingdb/htmlreport.py

```diff
@@ -679,11 +679,17 @@ def format_percent(num):
     return str(num) + '%'
 
 
-def first_decimal(number, digits=1):
-    """Return the first `digits` digit after the decimal point"""
-    number = abs(number)
-    number = number - int(number)
-    return int(round(number * 10**digits))
+def split_digits(number, digits_after_dp=1):
+    """Return digits before and after a decimal point"""
+    number = round(float(number), digits_after_dp)
+    return str(number).split('.')
+
+assert split_digits(1) == ['1', '0']
+assert split_digits(1.234) == ['1', '2']
+assert split_digits(1234.56) == ['1234', '6']
+assert split_digits(1234.56, 2) == ['1234', '56']
+assert split_digits(73.98) == ['74', '0']
+assert split_digits(-8.5) == ['-8', '5']
```

Comment:
Ad hoc tests. Works for me :+1:

Reply:
Yea, I figure we don't need a fancy runner, and they do always run this way.